driver-core/src/main/java/com/datastax/driver/core/ReplicationStategy.java

```diff
@@ -55,6 +56,7 @@ static ReplicationStrategy create(Map<String, String> replicationOptions) {
             }
         } catch (NumberFormatException e) {
             // Cassandra wouldn't let that pass in the first place so this really should never happen
+            logger.error("Failed to parse replication options: " + replicationOptions, e);
```

Comment:
I think it makes sense to leave the catch block.  While in practice it shouldn't happen as the values should be numeric, it doesn't hurt to catch the exception, since its otherwise an unchecked exception and in the event it does happen (for whatever reason) it'd be nice to handle and log it.